nodes/basic-pow/src/service.rs

```diff
@@ -1,5 +1,5 @@
 //! Service and ServiceFactory implementation. Specialized wrapper over substrate service.
-
+#![allow(clippy::needless_borrow)]
```

Comment:
Why do we need this? If it is in our code, we should remove the borrow. If this code comes from a Substrate macro, I'm fine with overriding adding this for now. But we should make an upstream PR to make the macro better. Here's an example of one I've done in the past https://github.com/paritytech/substrate/pull/9053